README.md

```diff
@@ -30,13 +31,18 @@ one you should use, refer to the [`argon2` repo][argon2].
 var argon2i = require('argon2-ffi').argon2i;
 // var argon2d = require('argon2-ffi').argon2d; if you'd like to use argon2d
 
-var password = new Buffer('password1');
-var salt = new Buffer('saltysalt');
-argon2i.hash(password, salt, function(err, res) {
-  console.log(res); // $argon2i$v=19$m=4096,t=3,p=1$c2FsdHlzYWx0$oG0js25z7kM30xSg9+nAKtU0hrPa0UnvRnqQRZXHCV8
+var password = 'password1'; // Can also be a Buffer
+crypto.randomBytes(128, function (err, salt) {
```

Comment:
I originally had it at 16, then changed it to 128 because that was the first google result. I hope you can tell I'm not a cryptographer :P. I've updated it to 32 bytes in the next push.